src/middlewares/commenter.js

```diff
@@ -4,34 +4,17 @@ import UserQuery from '../helpers/userQueries';
 const checkAuthorizedCommenter = async (req, res, next) => {
   const commenterId = req.user.id;
   const { requestId } = req.query;
-
   if (isNaN(requestId)) {
-    return res.status(422).json({
-      status: 422,
-      error: 'please provide the Id of the request, Id must be a number',
-    });
-  }
-
-  const requestInfo = await models.Request.findOne({
-    where: { id: requestId },
-  });
-
-  if (!requestInfo) {
-    return res.status(404).json({
-      status: 404,
-      error: 'request does not exist',
-    });
+    return res.status(422).json({ status: 422, error: 'please provide the Id of the request, Id must be a number' }); 
   }
+  const requestInfo = await models.Request.findOne({ where: { id: requestId } });
+  if (!requestInfo) return res.status(404).json({ status: 404, error: 'request does not exist' });
   const { requesterId } = requestInfo;
   const requester = await UserQuery.getUser('id', requesterId);
   const { lineManager } = requester.dataValues;
   const Manager = await UserQuery.getUser('email', lineManager);
```

Comment:
@Kagorora , is there a reason you capitalized this` manager` object?

Reply:
fixed as well, you are right only class name should be capitalized